### PR TITLE
Bug 1216919 - Fix test_browser_clear_browsing_history.py

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bab98e35f7d0397d06df957ecf01d19db64208f1

File: tests/python/gaia-ui-tests/gaiatest/apps/search/app.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def go_to_url(self, url):
             self._root_element = self.marionette.find_element(*self._browser_app_locator)
             self._root_element.find_element(*self._url_bar_locator).tap()
 
-        from gaiatest.apps.homescreen.regions.search_panel import SearchPanel
+        from gaiatest.apps.system.regions.search_panel import SearchPanel
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/search/app.py]: `apps/homescreen/app.py` contains another reference to `gaiatest.apps.homescreen.regions.search_panel`. Could you include it in your patch?

Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/search/app.py]: On the same way, we need to update the documentation:
* `tests/python/gaia-ui-tests/docs/apps/system.rst`
* `tests/python/gaia-ui-tests/docs/apps/homescreen.rst`

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/search/app.py]: `apps/homescreen/app.py` that import is going to be removed, it's not used anymore.

**Pull request merged**